src/clr-angular/wizard/providers/wizard-navigation.service.ts

```diff
@@ -737,45 +737,51 @@ export class WizardNavigationService implements OnDestroy {
 
     /**
      * @memberof WizardNavigationService
+     * @deprecated since 0.12
      */
-    private _wizardGhostPageState: string = GHOST_PAGE_ANIMATION.STATES.NO_PAGES;
+    private _wizardGhostPageStateDeprecated: string = GHOST_PAGE_ANIMATION_DEPRECATED.STATES.NO_PAGES;
```

Comment:
Keep the deprecation comments, revert the names throughout since they are private.